### PR TITLE
Solution DonchykD

### PR DESCRIPTION
**Pull request by DmytryTong**

### CODE REVIEW

**Review by SergiiMachulin — APPROVED** [review]
LGTM, but i suggest to add negative numbers check according to the checklist of README.md

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+            100,
+            [21, 17],
+            id="test should return result with large arguments"
+        )
+    ]
+)
+def test_of_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected_result: list) -> None:
```
Comment (Serhii-Leonenko) [inline app/test_main.py:50]: fix indentation

**Review by Serhii-Leonenko — CHANGES_REQUESTED** [review]
Agree with the previous comment, add a check for negative numbers, also I don't see tests for all age boundaries, what about it:

image

File: app/test_main.py (hunk 1 above)
Comment (DmytryTong) [reply app/test_main.py:50]: Fixed: negative number case and 27/27, 28/28 cases added. Identation - fixed.

**Review by YuriiKindrat — APPROVED** [review]
